src/test/java/com/eleks/academy/whoami/repository/GameInMemoryRepositoryTest.java

```diff
@@ -29,13 +28,14 @@ public void setMockMvc() {
 	}
 
 	@Test
-	void save() {
+	void saveTest() {
 		final String player = "player";
+
 		SynchronousGame game = new PersistentGame(player, gameRequest.getMaxPlayers());
 
 		assertThat(gameRepository.save(game))
-				.isEqualTo(gameRepository.save(new PersistentGame("player", 4)));
-
-		verify(gameRepository, times(2)).save(any(SynchronousGame.class));
+				.isEqualTo(new PersistentGame(player, 4));
+		assertThat(gameRepository.save(game).getPlayersInGame().get(0))
```

Comment:
please compare whole object, not just part of it

Reply:
Resolved